proposals/NNNN-update-published-app-services.md

```diff
@@ -0,0 +1,94 @@
+# Update Published App Services
+
+* Proposal: [SDL-NNNN](NNNN-update-published-app-services.md)
+* Author: [Jacob Keeler](https://github.com/jacobkeeler)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / Cloud Proxy / Policy Server / SHAID / RPC]
```

Comment:
```suggestion
* Impacted Platforms: [Core / iOS / Java Suite / Policy Server / SHAID / RPC]
```

As both the cloud proxy and android library now exist under Java Suite we'd like to use that as a platform name moving forward, thanks!